vitest.config.ts

```diff
@@ -7,4 +8,10 @@ export default defineConfig({
     include: ['./src/__tests__/**/*.test.jsx'],
     globals: true,
   },
+  resolve: {
+    alias: {
```

Comment:
@akartis-dev Why aren't we adding path in `tsconfig`

Reply:
I already add path in tsconfig, you can verify in changed file